setup.cfg

```diff
@@ -43,13 +43,13 @@ install_requires =
     deepmerge
     fs
     numpy
-    opentelemetry-api==1.9.0
-    opentelemetry-instrumentation==0.28b0
-    opentelemetry-instrumentation-aiohttp-client==0.28b0
-    opentelemetry-instrumentation-asgi==0.28b0
-    opentelemetry-sdk==1.9.0
-    opentelemetry-semantic-conventions==0.28b0
-    opentelemetry-util-http==0.28b0
+    opentelemetry-api>=1.9.0
```

Comment:
@bojiang what's the reason we lock the opentelemetry version before? was it to prevent installing the pre-release versions by accident?  if that's the case it should be ok to change this to `>=` now

Reply:
There are some issue with opentelemetry pre-release before, couldn't remember what the issues are. Idk if that got fixed upstream.

Reply:
We have to lock the version of otlp libs because they are not well organized.

Reply:
@aarnphm Unfortunately I needed to bump it since it was required by `opentelemetry-exporter-otlp`
@bojiang Would you be confortable if I lock them to their current version until the e2e tests are added? Although we don't have e2e tests, on our side we tested tracing with jaeger, zipkin and otlp exporters on these versions and didn't find any errors.